packages/compass-aggregations/src/components/aggregations/aggregations.jsx

```diff
@@ -3,7 +3,6 @@ import { connect } from 'react-redux';
 import Pipeline from '../pipeline';
 import { maxTimeMSChanged } from '../../modules/max-time-ms';
 import { collationStringChanged } from '../../modules/collation-string';
-import { toggleSample } from '../../modules/sample';
```

Comment:
Drive-by: this slice is not used by anything anymore, missed it in the previous clean-up